CMakeLists.txt

```diff
@@ -595,13 +600,15 @@ endif()
 # The avif_internal target should not be used by external code.
 if(BUILD_SHARED_LIBS)
     add_library(avif_internal STATIC ${AVIF_SRCS})
-    # Copy most properties from the public avif library target.
+    # Copy or duplicate most properties from the public avif library target.
     target_compile_definitions(avif_internal PRIVATE "$<TARGET_PROPERTY:avif,COMPILE_DEFINITIONS>")
     target_link_libraries(avif_internal PRIVATE "$<TARGET_PROPERTY:avif,LINK_LIBRARIES>")
-    target_include_directories(
-        avif_internal PUBLIC "$<TARGET_PROPERTY:avif,INTERFACE_INCLUDE_DIRECTORIES>"
-        PRIVATE "$<TARGET_PROPERTY:avif,INCLUDE_DIRECTORIES>"
-    )
+    target_include_directories(avif_internal PUBLIC ${libavif_SOURCE_DIR}/include)
+    target_include_directories(avif_internal PRIVATE ${AVIF_PLATFORM_INCLUDES} ${AVIF_CODEC_INCLUDES})
+    target_include_directories(avif_internal SYSTEM PRIVATE ${AVIF_PLATFORM_SYSTEM_INCLUDES} ${AVIF_CODEC_SYSTEM_INCLUDES})
+    if(NOT libyuv_FOUND)
+        target_include_directories(avif_internal PRIVATE ${libavif_SOURCE_DIR}/third_party/libyuv/include/)
+    endif()
```

Comment:
Lines 609-611 are new. Is this something that's missed in https://github.com/AOMediaCodec/libavif/pull/1534?

Reply:
These are the same as 587-589

Reply:
I'm sorry my previous comment was not clear. What I meant is: while the other changes in this pull request can be considered as modifications to existing code, only these three lines are additions. So I should have said "Lines 609-611 are additions." rather than "Lines 609-611 are new."

So my question was whether these three lines should have been part of https://github.com/AOMediaCodec/libavif/pull/1534.